packages/replayio/src/utils/initialization/promptForNpmUpdate.ts

```diff
@@ -0,0 +1,29 @@
+import { name } from "../../../package.json";
+import { prompt } from "../prompt/prompt";
+import { updateCachedPromptData } from "../prompt/updateCachedPromptData";
+import { highlight } from "../theme";
+import { UpdateCheckResult } from "./types";
+
+const PROMPT_ID = "npm-update";
+
+export async function promptForNpmUpdate(updateCheck: UpdateCheckResult<string>) {
+  const { fromVersion, toVersion } = updateCheck;
+
+  console.log("");
+  console.log("A new version of replayio is available!");
+  console.log("  Installed version:", highlight(fromVersion));
+  console.log("  New version:", highlight(toVersion));
+  console.log("");
+  console.log("To upgrade, run the following:");
+  console.log(highlight(`  npm install -g ${name}`));
+  console.log("");
+  console.log("Press any key to continue");
+  console.log("");
+
+  await prompt();
```

Comment:
One thing bothers me a little bit about this prompt - it doesn't work like the other one. it's just information and nothing happens when I confirm it. After seeing the other one I have a different expectation for this one - if I've seen one that runs the install process for me then I expect the same - familiar - process here. 

I know we might not want to run `npm install -g` for the user. They might be using the CLI through the per-project installation and they might be using an alternative package manager. Maybe we could try to detect those situations and offer the appropriate command (and actually run it)?

Reply:
I considered that, and I like the suggestion, but I'd rather leave it out of MVP (unless you feel strongly about it). I'll file a Linear issue though!